TrkHitReco/fcl/prolog.fcl

```diff
@@ -64,7 +65,7 @@ ScanTPClusterer : {
     MinHitError      : 5.0
     TimeRMS          : 2.0
     DeltaTimeBinMin  : 10
-    FilterAlgo       : 1
```

Comment:
Please add a comment which algorithm this corresponds to

Reply:
Done. 1 is for an algorithm that only considers the next neighbors, while 2 does a simple clustering. In practice, the results are quite similar.